src/sdk/netvars.hpp

```diff
@@ -111,4 +111,19 @@ namespace Offsets {
 }
 
 #define GETNETVAROFFSET(table, prop) Netvar::offsets.at({table, prop})
-#define NETVAR( table, prop, func, type ) type& func() {return *reinterpret_cast<type*>(uintptr_t(this) + GETNETVAROFFSET(table, prop));}; type* func##_ptr() {return reinterpret_cast<type*>(uintptr_t(this) + GETNETVAROFFSET(table, prop));}
+#define NETVAR(table, prop, func, type) type& func() {                                          \
+        return *reinterpret_cast<type*>(uintptr_t(this) + GETNETVAROFFSET(table, prop));        \
+    }                                                                                           \
+    void func##_set(type value) {                                                               \
+        if (!CONFIGBOOL("Misc>Misc>Misc>Disable Setting Netvars"))                              \
+            *(reinterpret_cast<type*>(uintptr_t(this) + GETNETVAROFFSET(table, prop))) = value; \
+    }                                                                                           \
+    type* func##_ptr() {                                                                        \
+        static bool warned = false;                                                             \
+        if (CONFIGBOOL("Misc>Misc>Misc>Disable Setting Netvars") && !warned) {                  \
+            warned = true;                                                                      \
+            std::cout << "Setting netvars is disabled, but i'm making a pointer to"             \
+              << prop "anyway, you better know what you're doing";                              \
```

Comment:
:concern:

Reply:
Like i said, idk what to do for viewangles, I could return nullptr but idk if crashing is a good idea

Reply:
Actually i meant because you are missing some "<<" there

Reply:
Well it works, strings concat anyway if you use them like that:
![image](https://user-images.githubusercontent.com/31628566/198874715-e0e9bc0a-cbdf-4eb1-85c2-b74f31c4a301.png)

Reply:
Yea you can concat strings like that everywhere, actually if you look at my source I heavily use this in inline asm, but I think it might be going against the code style the cheat already has

Reply:
Fair enough